### PR TITLE
create investor feed

### PR DESCRIPTION
**Pull request by mentalcaries**
A very basic start here, but it's the Investor feed for the start up cards.
This was mostly styling, and I am sure it can be refined further once we create the filters and cards

### CODE REVIEW

**Comment by vgalilei** [comment]
@mentalcaries Can you please name what part of the design do you implement or provide a screenshot of the current stage?

**Comment by mentalcaries** [comment]
It's the blank Investor/View startups page, which contains the filter and
the cards

[image: image.png]

File: frontend/src/components/investor-feed/InvestorFeed.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { ButtonText, FilterContainer, StartupContainer, StyledInvestorFeed } from './styledInvestorFeed';
+
+function InvestorFeed() {
+  return <div>
+    <StyledInvestorFeed>
+
+    <FilterContainer>
+      {/* button tags as placeholders */}
+    <button><ButtonText>Relevant | Others</ButtonText></button>
+    <button><ButtonText>Filter</ButtonText></button>
```
Comment (cjmaret) [inline frontend/src/components/investor-feed/InvestorFeed.js:12]: interesting wrapping the styled component in there. i notice we dont have a global style for buttons. maybe, we can ask, but in the meantime, this is probably fine

File: frontend/src/components/investor-feed/InvestorFeed.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { ButtonText, FilterContainer, StartupContainer, StyledInvestorFeed } from './styledInvestorFeed';
+
+function InvestorFeed() {
+  return <div>
+    <StyledInvestorFeed>
+
+    <FilterContainer>
```
Comment (cjmaret) [inline frontend/src/components/investor-feed/InvestorFeed.js:9]: i think the FilterContainer should be merged when its a finished product. It will probably take a bit of time, so you could consider removing it for this PR, or add in all the `select` and `range` inputs so it looks like the basic product on the figma

File: frontend/src/components/investor-feed/InvestorFeed.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      {/* button tags as placeholders */}
+    <button><ButtonText>Relevant | Others</ButtonText></button>
+    <button><ButtonText>Filter</ButtonText></button>       
+    </FilterContainer>
+    <StartupContainer>
```
Comment (cjmaret) [inline frontend/src/components/investor-feed/InvestorFeed.js:14]: perfect! man, empty lists are easy 😄

File: frontend/src/components/investor-feed/styledInvestorFeed.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+import { P } from '../global/textStyles';
+
+export const StyledInvestorFeed = styled.div`
```
Comment (cjmaret) [inline frontend/src/components/investor-feed/styledInvestorFeed.js:4]: cool with the styles! i've never done styled components before, this looks awesome

File: frontend/src/components/investor-feed/InvestorFeed.js (hunk 1 above)
Comment (mentalcaries) [reply frontend/src/components/investor-feed/InvestorFeed.js:12]: I tried to follow @mariawright05 's guide to styling but definitely open to improvement.

**Comment by cjmaret** [comment]
Hey Devin, good job on the styling here.. the only thing i want to mention is about the FilterContainer component (which is my second comment up there). Maybe we can hold off on it for now, unless you want to include the inputs

**Review by cjmaret — CHANGES_REQUESTED** [review]
Hey Devin, good job on the styling here.. the only thing i want to mention is about the FilterContainer component (which is my second comment up there). Maybe we can hold off on it for now, unless you want to include the inputs

File: frontend/src/components/investor-feed/InvestorFeed.js (hunk 2 above)
Comment (thinklikeadesigner) [reply frontend/src/components/investor-feed/InvestorFeed.js:9]: From the meeting this morning, I think @vgalilei wants all the components in the right positions on the page, even if it's just an empty box. This makes it so items have the correct layout and spacing on the page.

Comment (thinklikeadesigner) [reply frontend/src/components/investor-feed/InvestorFeed.js:9]: So for now, **relevant|others** should have the correct margins above **filter** (which is 20px) even if they aren't styled themselves. and **filter** should be 427x80 at 1440px as well as **relevant|others** and they should have a right margin of 40px so **list of startups** and **filter** should have a gap of 40px between them.

TLDR The position on the page is important, because other people will have to base the positioning of their elements off of yours. You can always style later, but shifting things around will shift everyone else's stuff too.

**Review by thinklikeadesigner — CHANGES_REQUESTED** [review]
Please add ```margin-bottom: 20px``` to **relevant|others** so that it matches the design

Screen Shot 2021-11-01 at 2 37 26 PM

Screen Shot 2021-11-01 at 2 37 53 PM

and the buttons should also be the same height and width as each other, since this will affect positioning of the start up list to the right
Screen Shot 2021-11-01 at 2 39 11 PM

This is more of a nit pick, but I think it would make more sense for just ``` <button><ButtonText>Filter</ButtonText></button>``` to be inside ``` <FilterContainer>```

Screen Shot 2021-11-01 at 2 43 21 PM

```<button><ButtonText>Relevant | Others</ButtonText></button>``` is a Switch toggle, so it can probably just float above ```<FilterContainer>``` 

Screen Shot 2021-11-01 at 2 45 31 PM

File: frontend/src/components/investor-feed/InvestorFeed.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+import { ButtonText, FilterContainer, StartupCardList, StyledInvestorFeed } from './styledInvestorFeed';
+
+function InvestorFeed() {
+  return <div>
+    <StyledInvestorFeed>
+
+    <FilterContainer>
+      {/* button tags as placeholders */}
+    <button><ButtonText>Relevant | Others</ButtonText></button>
```
Comment (thinklikeadesigner) [inline frontend/src/components/investor-feed/InvestorFeed.js:11]: Move this button outside of filter container, or rename filter container as "Wrapper"

Comment (thinklikeadesigner) [reply frontend/src/components/investor-feed/InvestorFeed.js:11]: button needs margin-bottom: 20px

**Review by thinklikeadesigner — CHANGES_REQUESTED** [review]

File: frontend/src/components/investor-feed/InvestorFeed.js (hunk 5 above)
Comment (mentalcaries) [reply frontend/src/components/investor-feed/InvestorFeed.js]: @thinklikeadesigner Agreed that it's a switch toggle. My rationale for putting it there was that it basically toggled the filter on and off. FilterContainer is basically supposed to contain all the filter-related content, but I can change it to Wrapper

**Review by thinklikeadesigner — APPROVED** [review]
